src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -70,9 +70,9 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args = null)
                 .ConfigureLogging((context, loggingBuilder) =>
                 {
                     loggingBuilder.ClearProviders();
-
                     loggingBuilder.AddDefaultWebJobsFilters();
                     loggingBuilder.AddWebJobsSystem<WebHostSystemLoggerProvider>();
+                    loggingBuilder.AddConsoleIfEnabled(context);
```

Comment:
@brettsam - Worker Logs are missing even though "fileLoggingMode" is set to "always"

Reply:
fileLoggingMode doesn't apply to this level. This change seems unrelated to the goal of the PR.

Reply:
Enabling console logging provided Worker start up logs.

Reply:
This is the WebHost - it shouldn't be logging to Console

Reply:
Will revert this. This was useful while running locally and development flag was enabled